### ISSUE TITLE
RayShapes in 3D sometimes causes player to vibrate in place

### ISSUE THREAD

**Issue by DenisBelmondo** [godotengine/godot#34663]
**Godot version:**
3.2

**OS/device including version:**
Windows 10 64-bit

**Issue description:**
Player with multiple RayShapes as part of their Collision Shapes vibrates in place very slightly when just on ground.

**Steps to reproduce:**
In the reproduction project, walk for a brief moment and then stop. You can observe the player vibrating in place very slightly even when fully stopped. This is very inconsistent, so it may take a few tries to be able to see it.

**Minimal reproduction project:**
[movement - Copy.zip](https://github.com/godotengine/godot/files/4007648/movement.-.Copy.zip)

**Additional notes:**
This still occurs even with one RayShape. To test, delete all the nodes named "LegX" (where X is a number) except for "Leg5".

**Comment by Calinou** [comment]
Can you reproduce this if you switch the physics engine to GodotPhysics in the Project Settings?

**Comment by DenisBelmondo** [comment]
Hello @Calinou, yes. I can verify that this occurs under Bullet and GodotPhysics on my end.

**Comment by jasonswearingen** [comment]
I am just starting to enter this hobby and godot, but this sounds pretty typical of any (all?) physics systems.   @DenisBelmondo perhaps try find a way to dampen physics on the object after stopping movement.

**Comment by DenisBelmondo** [comment]
Hello @jasonswearingen, welcome. Unfortunately, at least, as a result of my own testing, I have ensured that my player's Y velocity while standing on the floor is 0.0.

When explicitly setting the Y velocity to 0.0 during floor checks, the single RayShape setup seems to benefit the most (very little visible jittering in place), though the multiple RayShape setup seems to jitter regardless.

This could be because the use of multiple RayShapes in this way is non-standard. Many RayShapes at once trying to de-penetrate the player, or something along those lines.

**Comment by madmiraal** [comment]
Duplicate of #29392?

**Comment by David-Ochoa** [comment]
> Duplicate of #29392?

No, it's not the same issue. Because of [#29392](https://github.com/godotengine/godot/issues/29392) I had to change from inividual collision boxes to one big shape for the floor and I'm having the same problem: [Jitter](https://youtu.be/5pA7JvZEdio)
